.github/workflows/pr-rat-check.yml

```
@@ -38,3 +38,9 @@ jobs:
       - name: Run Apache RAT
         run: |
           java -jar apache-rat-0.16.1.jar -d . -E .rat-excludes | grep "== File:" && echo "The files listed above are missing license headers." && exit 1 || echo "All files have license headers."
+
+      - name: Check NOTICE Copyright year
+        run: |
+          (grep "Copyright $(date +%Y) The Apache Software Foundation" NOTICE && echo "NOTICE Copyright year is up to date") || (echo "NOTICE Copyright year is outdated" && exit 1)
```

Thank you for the PR.

```suggestion
          (grep "Copyright 2023-$(date +%Y) The Apache Software Foundation" NOTICE && echo "NOTICE Copyright year is up to date") || (echo "NOTICE Copyright year is outdated" && exit 1)
```
Based on the PJ Fanning's suggestion ( https://github.com/apache/incubator-kie-issues/issues/1618#issuecomment-2516665249 ), I have updated the year with range `2023-2024` ( https://github.com/apache/incubator-kie-kogito-runtimes/pull/3806 )

So, the grep would be like this. Thanks!